### PR TITLE
Add getStickerSet

### PR DESCRIPTION
**Pull request by LibertyLocked**
- [ ] All tests pass
- [x] I have run `npm run gen-doc`

### Description

Adds `getStickerSet` method

### References
https://core.telegram.org/bots/api#getstickerset

### CODE REVIEW

File: src/telegram.js [hunk 1]
```diff
@@ -1467,6 +1467,19 @@ class TelegramBot extends EventEmitter {
     form.ok = ok;
     return this._request('answerPreCheckoutQuery', { form });
   }
+
+  /**
+   * Use this method to get the sticker set given the name of the set.
+   * Returns a [StickerSet](https://core.telegram.org/bots/api#stickerset) object.
+   *
```
Comment (GochoMugo) [inline src/telegram.js]: Could you just copy the method description, as is, from the documentation.

File: src/telegram.js [hunk 2]
```diff
@@ -1467,6 +1467,19 @@ class TelegramBot extends EventEmitter {
...
+   * @param {String} name Name of the sticker set
```
Comment (GochoMugo) [inline src/telegram.js]: 2 spaces after `@param` please. It makes the type clauses e.g. `{String}` align perfectly with that of `@return`.

File: src/telegram.js [hunk 3]
```diff
@@ -1467,6 +1467,19 @@ class TelegramBot extends EventEmitter {
...
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
```
Comment (GochoMugo) [inline src/telegram.js]: Sorry, but could we use `@return` instead of `@returns`. We really like consistency. :smile:

Comment (GochoMugo) [inline src/telegram.js]: We are also missing the link to the method, using `@see`. Check out other methods for such links.

File: test/telegram.js [hunk 4]
```diff
@@ -26,6 +26,7 @@ if (!PROVIDER_TOKEN && !isCI) { // If is not running in Travis / Appveyor
 const USERID = process.env.TEST_USER_ID || 777000;
 const GROUPID = process.env.TEST_GROUP_ID || -1001075450562;
 const GAME_SHORT_NAME = process.env.TEST_GAME_SHORT_NAME || 'medusalab_test';
+const STICKER_SET_NAME = process.env.STICKER_SET || 'pusheen';
```
Comment (GochoMugo) [inline test/telegram.js]: We should probably document this on `test/README.md`.

**Comment by CapacitorSet** [comment]
Since it was abandoned, I took the liberty to fix it in #429. I hope the maintainer doesn't mind.

**Comment by GochoMugo** [comment]
Closing this as feature already addressed in PR #429.

Thanks @LibertyLocked for the PR.

**Pull request closed without merging**